src/components/pages/Login/index.tsx

```diff
@@ -112,33 +107,35 @@ const Login = () => {
                 </Typography>
                 <form onSubmit={handleSubmit}>
                     <Flex className={classes.inputWrapper} flexDirection="column">
-                        <TextField style={{ marginBottom: "15px"}} id="email-input" value={emailInput}onChange={handleInputChange("email")} type="email" label="E-Mail" variant="outlined" />
-                        <TextField style={{ marginBottom: "15px"}} id="password-input" value={passwordInput} onChange={handleInputChange("password")} type="password" label="Password" variant="outlined" />
+                        <TextField style={{ marginBottom: "15px" }} id="email-input" value={emailInput} onChange={handleInputChange("email")} type="email" label="E-Mail" variant="outlined" />
+                        <TextField style={{ marginBottom: "15px" }} id="password-input" value={passwordInput} onChange={handleInputChange("password")} type="password" label="Password" variant="outlined" />
                     </Flex>
 
-                    {errorMsg && <Alert style={{margin: "10px 0"}} severity="error">
+                    {errorMsg && <Alert style={{ margin: "10px 0" }} severity="error">
                         <AlertTitle>Upps...</AlertTitle>
                         {errorMsg}
                     </Alert>}
-                    {!isLoading ? null : 
-                    <Typography variant="subtitle1" paragraph>
-                        Loading...<CircularProgress/>
-                    </Typography>}
 
-                    <Flex justifyContent="space-between"> 
+                    <Flex justifyContent="space-between">
                         <Button onClick={handleBack} color="primary" type="button">Sign Up</Button>
                         <Fab color="primary" aria-label="add" type="submit">
-                            <Icon style={{color: "white"}}>keyboard_arrow_right</Icon>
+                            <Icon style={{ color: "white" }}>keyboard_arrow_right</Icon>
                         </Fab>
                     </Flex>
-                   
+
                 </form>
                 <Button onClick={handlePopupLogin} type="button">Login with Google!</Button>
             </div>
         </Flex>
         <Flex className={classes.footerWrapper} alignItems="center" justifyContent="flex-end" height="42px">
             <Typography align="right">Forgot password?</Typography>
         </Flex>
+
+        {!isLoading ? null :
```

Comment:
Here you could just do like the example below, but lets ignore that for now.
```suggestion
        { isLoading && <Backdrop ...
```